packages/zoe/src/contracts/coveredCall.js

```diff
@@ -16,42 +16,48 @@ import harden from '@agoric/harden';
 // extent of the invite:
 // { expirationDate, timerAuthority, underlyingAsset, strikePrice }
 
-import { makeHelpers } from './helpers/userFlow';
+import { makeZoeHelpers } from './helpers/zoeHelpers';
 
-export const makeContract = harden((zoe, terms) => {
-  const { rejectOffer, hasValidPayoutRules, swap } = makeHelpers(
-    zoe,
-    terms.issuers,
-  );
-  const ASSET_INDEX = 0;
-  const PRICE_INDEX = 1;
+export const makeContract = harden(zoe => {
+  const { swap, assertRoleNames, rejectIfNotOfferRules } = makeZoeHelpers(zoe);
+  assertRoleNames(harden(['UnderlyingAsset', 'StrikePrice']));
 
   const makeCallOptionInvite = sellerHandle => {
     const seat = harden({
-      exercise: () =>
-        swap(sellerHandle, inviteHandle, `The covered call option is expired.`),
+      exercise: () => {
+        const expected = harden({
```

Comment:
Can this be `expectedRules`?

Does this specify that no exit rule is allowed or that any rule is accepted?

Reply:
Any rule is accepted. RejectIf does nothing if no `expected` are passed